tokio/tests/task_local_set.rs

```diff
@@ -566,6 +566,48 @@ async fn spawn_wakes_localset() {
     }
 }
 
+#[test]
+fn store_local_set_in_thread_local_with_runtime() {
```

Comment:
Link to issue on this test?

Reply:
Is there a relevant discussion? We don't *usually* link to issues afaik unless there is key context.

Reply:
I guess no.